dNet/ClientPackets.cpp

```diff
@@ -284,17 +287,77 @@ void ClientPackets::HandleChatModerationRequest(const SystemAddress& sysAddr, Pa
 		receiver.push_back(static_cast<uint8_t>(character));
 	}
 
+	if (!receiver.empty()) {
+		if (std::string(receiver.c_str(), 4) == "[GM]") {
+			receiver = std::string(receiver.c_str() + 4, receiver.size() - 4);
+		}
+	}
+
```

Comment:
This is a bit confusing to read at first glance - I would recommend using .substr here instead to clarify what specifically is going on here instead of this method

Reply:
I will add comments